sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -278,7 +278,10 @@ def inplace_csr_row_normalize_l1(X):
     _inplace_csr_row_normalize_l1(X.data, X.shape, X.indices, X.indptr)
 
 
-def _inplace_csr_row_normalize_l1(np.ndarray[floating, ndim=1] X_data, shape,
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
```

Comment:
Thanks for @jnothman 's explanation, Cool!

@rvraghav93 I add it partially just because original code has these.
But for @cython.cdivision(True), you can see [this](https://github.com/cython/cython/wiki/enhancements-division)